src/main/kotlin/io/embrace/android/intellij/plugin/network/HttpClient.kt

```diff
@@ -0,0 +1,39 @@
+package io.embrace.android.intellij.plugin.network
+
+import java.io.IOException
+import java.net.URI
+import java.net.URISyntaxException
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+class HttpClient : ApiClient {
```

Comment:
I think we should use another name so it doesn't create confusion with java.net.http.HttpClient